### ISSUE TITLE
ES-FR-IT Translations Updates January 2023

### ISSUE THREAD

**Issue by wocmultimedia** [publishpress/PublishPress-Authors#1177]
Hi Steve,
Here's the reviewed and updated ES-FR-IT translations of Authors free, January 2023. Please,  add to plugin languages folder.
**Please, have also a check because the file .pot contained in the languages folder is obsolete with fewer strings than actual have.**

[Authors_ES-FR-IT_translationUpdates_January2023.zip](https://github.com/publishpress/PublishPress-Authors/files/10413732/Authors_ES-FR-IT_translationUpdates_January2023.zip)

Thanks,
Angelo

**Comment by ojopaul** [comment]
@stevejburge @wocmultimedia 

I refresh translation before every release and did same before the lats release.

I'm wondering how you confirmed it was outdated?

**Comment by wocmultimedia** [comment]
Hi,
Situation in my folder. I have no pleasure to state something not true.
![FilePot_insideAuthorsFolder](https://user-images.githubusercontent.com/2352240/212640593-abe0f546-795c-4527-b44e-74aab199bf90.jpg)
![IT_FilePo_insideAuthorsFolder](https://user-images.githubusercontent.com/2352240/212640597-ed29d925-e9a9-43ae-8fa3-3374be8b8f3a.jpg)
Maybe is my installation but if I update like any other user this is my situation

**Comment by ojopaul** [comment]
@wocmultimedia 

Nobody is insisting on you reporting something that's not true. In situation like this, anything could be responsible and since i re-downloaded the distributed version on wordpress.org to confirm but on synching the .pot file, i still see no new string, it's normal for me to ask how you confirmed this issue so i can look at it.

Now, back to the main issue, it's normal to report difference in the issue but not always all the time is the one with more strings always the updated one.

Like in the issue, you can refer to the attached screenshot below this comment and you'll see i just synch again and the correct strings is 562 which shows the **IT** translation contain obsolete strings that has been removed and reason it has most translations.

If there's other ways you really confirm the files in pot translation is missing strings that exist in the plugin after checking, i'm open to looking it up again.

![update](https://user-images.githubusercontent.com/92122385/212643126-a0c6a2dd-3bea-46ea-9224-5c2562da2f7f.jpg)

**Comment by wocmultimedia** [comment]
Hi,
First of all sorry if my reply was creating a not wanted kind of dispute. I just would like to know ho do I have to proceed with translation. Because if I update the plugin, whatever it is, which file I have to consider correct? I have been checking the different plugins just in this moment and I found that in some case Pro and Free have the same po/mo files while in some Pro the pot is missing at all. 
Please, let me know your way of working, so I can provide a correct updated translation for each language and plugin version.
It's only matter of finding an agreement on the working process and nothing else. Thanks a lot for understanding my difficulties.
Angelo

**Comment by ojopaul** [comment]
It's fine @wocmultimedia 

Will this work, if i updated every translation before releases instead of just the pot file?

That way, obsolete strings or updated strings will always be same and all you just need to do is update?

**Comment by wocmultimedia** [comment]
Hi @ojopaul,
That's the best way if you can update directly the single po language. You already did with some of this and in fact you had already did it with Gutenberg. Yes, it helps better than only refreshing the pot file.
Please, let me know if something is wrong in my translations because I just used the refreshed po or I refreshed with the existing  pot file.
Thanks a lot
Angelo

**Comment by ojopaul** [comment]
That solves the issue @wocmultimedia

I'll start doing that from the next release that should be going out today.

Thanks

**Comment by wocmultimedia** [comment]
Thanks a lot! Please, have a look to languages folder of Taxo/Simple tags because I found that my local installation has po/pot files with 763 strings while the online in WP-system has 922

**Comment by ojopaul** [comment]
All right, understood. @wocmultimedia

**Comment by wocmultimedia** [comment]
Hi,
We are cleaning what have been let not checked for several version, so far. Often my confusion is because files in WP-System appears different but with the new agreement I will know that what do you set in the plugins folder is the right one.
Thanks

**Comment by wocmultimedia** [comment]
Just to give a better idea. In my local installation of Publishpress (now planner) the file pot has 640 strings, the French po files 722 while the WP-System show 755 strings of which only 710 translated. 
So if I download the wp-system po file to make the new translation starts the confusion, because if I don't download the file from wp it would appears not full translated

**Comment by wocmultimedia** [comment]
OK! Perfect!

**Comment by wocmultimedia** [comment]
Hi @ojopaul @stevejburge 
I have a simple question.
According to this morning agreement the file inside the new version of Authors languages folder have been refreshed and both file pot and single po language file show 562 strings with 2 strings to be translated. Fine!

**Comment by wocmultimedia** [comment]
Hi @ojopaul @stevejburge
This morning I translated in Italian the two strings untranslated inside the local folder of Authors free and when I tried to upload in the WP-System they set correctly in place. I'm astonished and dazzled because the local strings are 562 and the WP-Systems shows 627 strings.
So according to our agreement I will translate what I find inside the plugin local folder and then I try to upload in the WP-System, eventually I will make some retouch to strings coming from WP-system.

Is it ok for you?
Waiting for a quick reply.
Angelo

**Comment by ojopaul** [comment]
Hello @wocmultimedia 

Thanks for the update and glad it works.

I personally don't know much about how WordPress translation system works but YES, translate what you find inside plugin as that's the updated one.

Maybe WordPress doesn't remove translated strings or it probably take time to reflect.

Thanks

**Comment by wocmultimedia** [comment]
Hi @ojopaul
Wen a common user updates the plugin it receive the plugin folder complete of the languages one but also WP-system ask to update translation. Latest one goes in 
LocalDisc:\Bitnami\wordpress-5.5.3-0\apps\wordpress\htdocs\**wp-content\languages\plugins** this translation often override the one in plugin languages folder. So in any case we have to make sure that both translations are updated.

This doesn't apply to Pro version that of course uses only the translations inside local folder.

So I will take care of both situation. I suppose that WP-System add something that is inside another part of the plugin such a dist folder. It happened with another plugin I translated for a new client until we discovered that was WP-system that joined different folder strings.
Thanks for this precious collaboration
Have a nice day
Angelo
